docs/docs/features/key-assignment.md

```diff
@@ -41,7 +41,7 @@ consumerd tendermint show-validator # {"@type":"/cosmos.crypto.ed25519.PubKey","
 Then, make an `assign-consensus-key` transaction on the provider chain in order to inform the provider chain about the consensus key you will be using for a specific consumer chain.
 
 ```
-gaiad tx provider assign-consensus-key <consumer-chain-id> '<pubkey>' --from <tx-signer> --home <home_dir> --gas 900000 -b block -y -o json
+gaiad tx provider assign-consensus-key <consumer-chain-id> '<pubkey>' --from <tx-signer> --home <home_dir> --gas 900000 -b sync -y -o json
```

Comment:
Oh, did they change the flag in 47?

Reply:
[deprecated](https://github.com/cosmos/cosmos-sdk/blob/release/v0.47.x/UPGRADING.md#broadcast-mode)